dev-docs/bidders/smartytech.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: bidder
+title: SmartyTech
+description: Prebid SmartyTech Bidder Adaptor
+pbjs: true
+biddercode: smartytech
+media_types: banner
```

Comment:
You may want to add as much fields as possible. See https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter

This helps sales houses and publishers to vet your adapter upfront.